### DISCUSSION TITLE
❓ Questions:how to convert the measurement?

### DISCUSSION THREAD

**Discussion by lfgogogo** [snakers4/silero-vad#70] [Q&A] [answered]
The measurement of the results is sample,now i want to convert it to second.Maybe divide 16000 into the results?I am not sure about it,please help,thanks.

**Comment by snakers4** [comment, marked as answer]
I do not remember how the 8 kHz model behaves (whether it resamples to 16, just test), but the conversion of samples to seconds is straightforward given the sampling rate.
